web/rootfs/defaults/settings-config.js

```diff
@@ -377,9 +378,26 @@ config.hiddenPremeetingButtons = [ '{{ join "','" (splitList "," .Env.HIDE_PREME
 
 // Configure remote participant video menu
 if (!config.hasOwnProperty('remoteVideoMenu')) config.remoteVideoMenu = {};
+config.remoteVideoMenu.disabled = {{ $DISABLE_REMOTE_VIDEO_MENU }};
 {{ if .Env.DISABLE_KICKOUT -}}
 config.remoteVideoMenu.disableKick = {{ .Env.DISABLE_KICKOUT }};
 {{ end -}}
 {{ if .Env.DISABLE_GRANT_MODERATOR -}}
 config.remoteVideoMenu.disableGrantModerator = {{ .Env.DISABLE_GRANT_MODERATOR }};
-{{ end -}}
\ No newline at end of file
+{{ end -}}
+{{ if .Env.DISABLE_PRIVATE_CHAT -}}
```

Comment:
While you're here, can you please set these options unconditionally, with defaults?

Reply:
Added a defaults like `{{ $DISABLE_PRIVATE_CHAT:= .Env.DISABLE_PRIVATE_CHAT | default "false" | toBool -}}`
And now using `config.remoteVideoMenu.disablePrivateChat = {{ $DISABLE_PRIVATE_CHAT }};`